src/notation/qml/MuseScore/NotationScene/qmldir

```diff
@@ -9,3 +9,4 @@ ConcertPitchControl 1.0 ConcertPitchControl.qml
 NoteInputBar 1.0 NoteInputBar.qml
 NoteInputBarCustomizationDialog 1.0 NoteInputBarCustomizationDialog.qml
 UndoRedoToolBar 1.0 UndoRedoToolBar.qml
+TempoView 1.0 TempoView.qml
```

Comment:
why you are exporting this component from two modules(CommonScene, Notation)?

Reply:
removed